src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,48 @@
+import classNames from 'classnames';
+import { FilterStatus } from '../../types/Filter';
+
+type Props = {
+  filterStatus: FilterStatus
+  onFilter: (filterStatus: FilterStatus) => void;
+};
+
+export const Filter: React.FC<Props> = ({ filterStatus, onFilter }) => {
+  return (
+    <nav className="filter" data-cy="Filter">
+      <a
+        data-cy="FilterLinkAll"
+        href="#/"
+        className={classNames(
+          'filter__link',
+          { selected: filterStatus === 'all' },
+        )}
+        onClick={() => onFilter('all')}
```

Comment:
Will be better 
```
onClick={() => (
      filterStatus !== 'all'
        && onFilter('all')
   )}
```
In this case `onFilter` won't work if `filterStatus` didn't change